cypress/integration/ete/reset_password.spec.js

```diff
@@ -4,8 +4,8 @@ describe('Password reset flow', () => {
   context('Account exists', () => {
     // This test depends on this Mailslurp account already being registered
     const existing = {
-      email: '0298a96c-1028-4e3d-b943-a2b478c84dbd@mailslurp.com',
```

Comment:
Should this key not be private ? Can we put it in config instead ?

Reply:
good point! this is how the emails were set up by @oliverlloyd so not sure if he considered this already. I guess we could have a test email config file that we populate?

Reply:
seems important because someone could come along and use up our mailbox limit

Reply:
This isn't the key. It's just a random uuid to get a unique email address

Reply:
The key (the api key) is set as a Github Secret

Reply:
> seems important because someone could come along and use up our mailbox limit

This is technically true so we could abstract it... That said, I suspect this vector is wide open is many other places (do we keep test email accounts secret?) and any attacker would lose more than they gain, so it would purely be an attack of spite. But if we really to be sure we could put these in env vars.

The downside to this is it adds to the work needed to onboard new devs, adding new tests and changing values. Plus the _whole team_ would need to action every change/new test so that they also have the secret values set locally. Personally I don't think this is worth it but it's a judgement call.